File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntDeployVar.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumStudioVarKind;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntDeployVar extends StudioBase
{
  public EnumStudioVarKind kind;

  public MetaIdVar metaId;

  @Nullable
  public Object varValue;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntDeployVar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import kotlinx.serialization.json.JsonElement
import com.neome.api.meta.base.Types.EnumStudioVarKind
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase

interface StudioEntDeployVar : StudioBase
{
  val kind: EnumStudioVarKind
  val metaId: MetaIdVar
  val varValue: JsonElement?
}